### ISSUE TITLE
Getting Started tutorial leads to errors

### ISSUE THREAD

**Issue by Tylersuard** [google-coral/tflite#4]
python3 classify_image.py \
--model models/mobilenet_v2_1.0_224_inat_bird_quant_edgetpu.tflite \
--labels models/inat_bird_labels.txt \
--input images/parrot.jpg

Response:
AttributeError: 'Delegate' object has no attribute '_library'

**Comment by Tylersuard** [comment]
Using Raspberry Pi 4B

**Comment by Namburger** [comment]
@Tylersuard could you check if you system can recognize the USB accelerator?

**Comment by inktomi** [comment]
I get a similar error trying to run the sample on a PixelBook. 

```Traceback (most recent call last):
  File "/home/.../.local/lib/python3.5/site-packages/tflite_runtime/interpreter.py", line 161, in load_delegate
    delegate = Delegate(library, options)
  File "/home/.../.local/lib/python3.5/site-packages/tflite_runtime/interpreter.py", line 120, in __init__
    raise ValueError(capture.message)
ValueError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "classify_image.py", line 122, in <module>
    main()
  File "classify_image.py", line 99, in main
    interpreter = make_interpreter(args.model)
  File "classify_image.py", line 73, in make_interpreter
    {'device': device[0]} if device else {})
  File "/home/..../.local/lib/python3.5/site-packages/tflite_runtime/interpreter.py", line 164, in load_delegate
    library, str(e)))
ValueError: Failed to load delegate from libedgetpu.so.1

**Comment by matt-hewett** [comment]
Hi, I had a similar error following the 'getting started' guide on a Dell Inspiron 15 7000 running Ubuntu 18.04. I found the issue to be with the standard frequency libedgetpu1-std library installed in step 1.2 of the guide. My observations were:

After following the guide and attempting to run the classify_image.py I got the following error:

python3 classify_image.py --model models/mobilenet_v2_1.0_224_inat_bird_quant_edgetpu.tflite --input images/parrot.jpg --labels models/inat_bird_labels.txt 
Traceback (most recent call last):
  File "classify_image.py", line 122, in <module>
    main()
  File "classify_image.py", line 99, in main
    interpreter = make_interpreter(args.model)
  File "classify_image.py", line 73, in make_interpreter
    {'device': device[0]} if device else {})
  File "/home/matt/.local/lib/python3.6/site-packages/tflite_runtime/interpreter.py", line 161, in load_delegate
    delegate = Delegate(library, options)
  File "/home/matt/.local/lib/python3.6/site-packages/tflite_runtime/interpreter.py", line 90, in __init__
    self._library = ctypes.pydll.LoadLibrary(library)
  File "/usr/lib/python3.6/ctypes/__init__.py", line 426, in LoadLibrary
    return self._dlltype(name)
  File "/usr/lib/python3.6/ctypes/__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libedgetpu.so.1: cannot open shared object file: No such file or directory
Exception ignored in: <bound method Delegate.__del__ of <tflite_runtime.interpreter.Delegate object at 0x7f5268e60ef0>>
Traceback (most recent call last):
  File "/home/matt/.local/lib/python3.6/site-packages/tflite_runtime/interpreter.py", line 125, in __del__
    if self._library is not None:
AttributeError: 'Delegate' object has no attribute '_library'

---------------------------------------------------------------------------

Notice this error: **OSError: libedgetpu.so.1: cannot open shared object file: No such file or directory**

I entered the command _ls /usr/lib/x86_64-linux-gnu/libedgetpu*_ to check if the libedgetpu.so.1 shared object file was present. It was not.

So, I went back to the guide and installed the max frequency edge tpu runtime option instead. I then ran the above command again and the libedgetpu.so.1 shared object file was now present.

I then ran the classify_image.py script successfully.

**Comment by Namburger** [comment]
@matt-hewett weird, both library should be installed to the same directory 👀
Did you use the install.sh script or the apt-get?

**Comment by matt-hewett** [comment]
@Namburger I used apt-get. I agree that it's weird. I just double checked by uninstalling the max frequency edge tpu runtime using _sudo apt-get remove libedgetpu1-max --purge_ so that the libedgetpu.so.1 shared object file was removed from my _/usr/lib/x86_64-linux-gnu/_ directory. I then reinstalled using the standard edge tpu runtime version via _sudo apt-get install libedgetpu1-std_. After this the libedgetpu.so.1 shared object file was in my _/usr/lib/x86_64-linux-gnu/_ directory as is expected.

As to why libedgetpu.so.1 didn't install correctly the first time I have no idea.
